GitVersion.yml

```
@@ -8,6 +8,12 @@ branches:
     prevent-increment:
       when-current-commit-tagged: true
     source-branches: []
+  hotfix:
+      regex: hotfix[/-]
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)